app/graphql/mutations/profile_link/delete.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class Mutations::ProfileLink::Delete < Mutations::Base
+  include FancyMutation
+
+  description 'Delete a profile link'
+
+  input do
+    argument :profile_link_id, ID,
+      required: true,
+      description: 'The profile link to delete'
```

Comment:
Using id here vs ProfileLinkSite in update makes me think we should just choose one or the other to "identify" the link. I'm kinda favoring just ProfileLinkSite, so that clients don't have to find the ID before deletion